input_parser.star

```diff
@@ -23,13 +23,16 @@ DEFAULT_DEPLOYMENT_STAGES = {
     # Deploy cdk-erigon node.
     # TODO: Remove this parameter to incorporate cdk-erigon inside the central environment.
     "deploy_cdk_erigon_node": True,
+    # Deploy contracts on L2 (as well as fund accounts).
+    "deploy_l2_contracts": True,
 }
 
 DEFAULT_IMAGES = {
     "agglayer_image": "ghcr.io/agglayer/agglayer:feature-storage-adding-epoch-packing",  # https://github.com/agglayer/agglayer/pkgs/container/agglayer-rs
     "cdk_erigon_node_image": "hermeznetwork/cdk-erigon:v2.1.0",  # https://hub.docker.com/r/hermeznetwork/cdk-erigon/tags
     "cdk_node_image": "ghcr.io/0xpolygon/cdk:0.3.0",  # https://github.com/0xpolygon/cdk/pkgs/container/cdk
     "cdk_validium_node_image": "0xpolygon/cdk-validium-node:0.7.0-cdk",  # https://hub.docker.com/r/0xpolygon/cdk-validium-node/tags
+    "l2_contracts_image": "minhdvu/toolbox:0.0.7",  # https://hub.docker.com/r/leovct/toolbox/tags
```

Comment:
@leovct I'll update this if you think the image looks good and when you push the new tag to your account.

Reply:
You build it using latest commit of polycli?

Reply:
there have been new commits since then, but latest will work

Reply:
ah nevermind, hold off on this for now, it may not be needed

Reply:
I'm working on a small script to automate this image build process btw, soon you won't need me :)